README.md

```diff
@@ -144,19 +143,23 @@ The string representations of each of these objects are appended together in the
 
 see https://developer.mozilla.org/en-US/docs/Web/API/console for more information about console logging.
 
-### SetTimeout
+### Timers

```

Comment:
I think this is from a another PR, as well as all the other time changes?

Reply:
See above

Reply:
Done